README.md

```diff
@@ -6,69 +6,89 @@ Manager for Longhorn.
 
 ## Requirement
 
-1. Existing Kubernetes Cluster 1.16+.
-2. Make sure `iscsiadm`/`open-iscsi` has been installed on the host.
+- Existing Kubernetes Cluster 1.21+.
+- [Mount propagation](https://kubernetes-csi.github.io/docs/deploying.html#enabling-mount-propagation) must be enabled.
+- Make sure `iscsiadm`/`open-iscsi` and `nfs-common`/`nfs-utils`/`nfs-client` has been installed on the host.
+  - Recommend that you execute the [environment_check.sh](https://raw.githubusercontent.com/longhorn/longhorn/master/scripts/environment_check.sh) script to check if all requirements have been fulfilled.
+- The host filesystem supports the `file extents` feature to store the data. Currently we support:
+  - ext4
+  - XFS
+
+For more details of the installation requirements, please refer to the [Longhorn documentation.](https://longhorn.io/docs/latest/deploy/install/#installation-requirements)
 
 ## Build
 
 `make`
 
 ## Deployment
 
-`kubectl create -Rf deploy/install`
+- `kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn.yaml`
 
 It will deploy the following components in the `longhorn-system` namespace:
-1. Longhorn Manager
-2. Longhorn Flexvolume Driver for Kubernetes
-3. Longhorn UI
+
+- Longhorn Managers
+- Longhorn Flexvolume Driver for Kubernetes
```

Comment:
`Longhorn Flexvolume Driver for Kubernetes` is not supported, right?

Reply:
Removed. Thanks.